### PR TITLE
RA-1476: Upgrade new Refapp from Bootstrap 3 to 4

### PR DESCRIPTION
**Pull request by Charpell**
## JIRA TICKET NAME
[RA-1476: Setup testing environment with Jest ](https://issues.openmrs.org/browse/RA-1476)

## SUMMARY:
Currently the new refapp uses Bootstrap 3. Here are few reasons for upgrading

1 Bootstrap now uses Sass instead of LESS.

2 Support for Flexbox. Now developers have the option to choose Flexbox based grid system.

3 All the JavaScript plugins have been rewritten to support ES6. This means developers have better plugin support in Bootstrap 4 and can take advantage of the new ES6 features.

4 The responsive typography is way easier in Bootstrap 4. This is because the new Bootstrap has moved from pixels to rem and em units, which is a good thing.

## Issue I worked on
[RA-1476](https://issues.openmrs.org/browse/RA-1476)

## Checklist: I completed these to help reviewers
 - [x] My pull request only contains ONE single commit (the number above, next to the 'Commits' tab is 1).
No? -> [read here](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) on how to squash multiple commits into one

- [x] My IDE is configured to follow the [code style](https://wiki.openmrs.org/display/docs/Java+Conventions) of this project.
No? Unsure? -> [configure your IDE](https://wiki.openmrs.org/display/docs/How-To+Setup+And+Use+Your+IDE), format the code and add the changes with `git add . && git commit --amend`

 - [ ] I have **added tests** to cover my changes. (If you refactored existing code that was well tested you do not have to add tests)
No? -> write tests and add them to this commit `git add . && git commit --amend`

- [x] I ran `mvn clean package` right before creating this pull request and added all formatting changes to my commit.
No? -> execute above command

- [ ] All new and existing **tests passed**.
No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.

 - [x] My pull request is **based on the latest changes** of the master branch.
No? Unsure? -> execute command `git pull --rebase upstream master`

### CODE REVIEW

File: postcss.config.js [hunk 1]
```diff
@@ -0,0 +1 @@
+module.exports = {};
```
Comment (humanityjs) [inline postcss.config.js]: Add a new line at the end of the file

Comment (Charpell) [reply postcss.config.js]: I have updated my PR

Comment (humanityjs) [reply postcss.config.js]: 👍

**Comment by humanityjs** [comment]
Looks good 👍

**Pull request merged**